go.sum

```diff
@@ -24,8 +24,11 @@ github.com/stretchr/testify v1.4.0/go.mod h1:j7eGeouHqKxXV5pUuKE4zz7dFj8WfuZ+81P
 golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd/go.mod h1:CfG3xpIq0wQ8r1q4Su4UZFWDARRcnwPjda9FqA0JpMk=
 golang.org/x/net v0.0.0-20220809184613-07c6da5e1ced h1:3dYNDff0VT5xj+mbj2XucFst9WKk6PdGOrb9n+SbIvw=
 golang.org/x/net v0.0.0-20220809184613-07c6da5e1ced/go.mod h1:YDH+HFinaLZZlnHAfSS6ZXJJ9M9t4Dl22yv3iI2vPwk=
```

Comment:
@dgtm just want to confirm - did you run `go mod tidy`? Not sure if these older versions should be removed from `go.sum` or not

Reply:
Good call. I ran `go mod tidy` and it removed the unnecessary lines.

Reply:
@willarmiros are you planning to cut a release anytime soon?

Reply:
@dgtm This fix has been released to Dockerhub & ECR public. v3.3.5 of the daemon will be available from S3 in all regions shortly, and we will cut a release when that is the case.